x-pack/legacy/plugins/uptime/public/components/higher_order/responsive_wrapper.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPanel } from '@elastic/eui';
+import React, { FC } from 'react';
+import styled from 'styled-components';
+
+const ResponsiveWrapper = styled.div`
+  margin-left: 120px;
```

Comment:
So, is there no EUI way to get this to work in a responsive way?

Reply:
You can see [this comment](https://github.com/elastic/kibana/pull/47176#issuecomment-539910050) for more info about why we have this custom styling here. I can ask someone from EUI to review this as well to see if they have a better recommendation.

Reply:
For more clarification, we wanted to introduce additional whitespace to balance out the view a bit, and because there wasn't enough separation in the default mobile view. In essence we're still using EUI's styling, just customizing it slightly. I've asked the EUI team to give us feedback on this though.